### DISCUSSION TITLE
Another thanks :-)

### DISCUSSION THREAD

**Discussion by grandaspanna** [SpenceKonde/megaTinyCore#364] [Show and tell]
Also wanted to say thanks for making my small projects possible and practical. These little 1-series units are so cool and the core has made it so easy to program. Here's a little 1616 project (afaik - it's the smallest form factor package) and programming is done with a small clip to the round pad in the corner: clip on, burn, upload, all done.
![IMG_5605](https://user-images.githubusercontent.com/54263749/111302394-75444180-86a7-11eb-8679-d05c5527cd0a.jpg)

**Comment by SpenceKonde** [comment]
Nice, glad to hear it! And yah 0 I think that is the second smallest package that an AVR is available in. The rather dismal 1k flash-equipped ATTiny10 comes in SOT-23-6. I don't understand why they couldn't take a tiny412 and put it the SOT-23-sized package with 8 pins (one does exist). Is the die really too big for that??

If you could, say. get the power and ground lines and UPDI line on 0.1" spacing, could use appropriate pogopins to program even easier. (I'm going to start selling 1x3p pogopins housings soon - only detail is that I was off by a few hundredths of a mm in a critical dimension, so the holes don't hold them in place.

**Reply by grandaspanna** [reply]
My biggest disappointment (technically one of many) is that there is now zero supply of the 1616 worldwide. Looks like another production run is still many weeks away. Close behind is the lack of an "812". That in an MSOP package would be very cool. 8-pin 2-series and Dx... ;-)

**Comment by SpenceKonde** [comment]
The 1626's are available though, and in that same microscopic package... 

I would venture a guess that the 2-series rampup (as well as the well documented worldwide semiconductor fab shortage that's been causing stuff like automotive manufacturers to curtail production because thy can't get the chips for the cars' computers) was a big part of the reason for that....
